src/java/org/apache/cassandra/config/EncryptionOptions.java

```diff
@@ -280,13 +340,14 @@ public int hashCode()
 
         public ServerEncryptionOptions()
         {
+            unsetEnabled(); // set to null DatabaseDescriptor can check if it was explicitly set.
```

Comment:
is this needed?  the default is null so setting null shouldn't be required.

Reply:
Agreed -- not needed.

Reply:
So removing in the current form causes test failures as the Yaml config loader I was originally using initializes the enabled field.

```
Caused by: org.apache.cassandra.exceptions.ConfigurationException: enabled should not be configured for server_encryption_options, must use internode_encryption
	at org.apache.cassandra.config.EncryptionOptions$ServerEncryptionOptions.tlsEncryptionPolicy(EncryptionOptions.java:365)
	at org.apache.cassandra.security.SSLFactory.validateSslCerts(SSLFactory.java:409)
	... 12 more
```

I'll try with the improved version.